hack/testdata/rollingupdate-statefulset.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1beta1
```

Comment:
I filed https://github.com/kubernetes/kubernetes/pull/51061 for manifests other than StatefulSet